tests/ts_A_create_table_not_null_pk_fields.erl

```diff
@@ -0,0 +1,37 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ts_A_create_table_not_null_pk_fields).
+
+-behavior(riak_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-export([
+         confirm/0
+        ]).
+
+confirm() ->
+    ClusterType = single,
+    DDL = ts_util:get_ddl(not_null_primary_key_field_fail),
```

Comment:
It would make sense to do this globally, for all ts_* modules. @javajolt if you are ok with this, I undertake to do this (in this PR).